build-support/migration-support/migrate_to_toml_config.py

```diff
@@ -14,6 +14,8 @@
 from pathlib import Path
```

Comment:
Everything in this folder should not change. It's important that it does not use any pants code. They are standalone strips